README.md

```diff
@@ -2,14 +2,14 @@
 PathwaysJob API is an OSS Kubernetes-native API, to deploy ML training and batch inference workloads, using Pathways on GKE. 
 //ToDo(roshanin) - add intro for Pathways.
 ## Description
-The PathwaysJob is an API that provides an easy way to run JAX workloads using Pathways. It support two modes of deployment.
+The PathwaysJob is an API that provides an easy way to run JAX workloads using Pathways. It support two modes of deployment. A PathwaysJob instance bundles the Pathways resource manager(RM) AKA Pathways server, the Pathways proxy server and the user workload containers into a single pod called "pathways-head". When the user pod is not provided (headless workloads), the "pathways-head" pod consists of the Pathways RM and the proxy server.
 ### Colocate mode
-The 'colocate' mode bundles the Pathways resource manager(RM), the Pathways proxy and the user workload containers into a single pod called "leader" and deploys them besides a "worker" pod on one of the TPU workers. This is preferred for Pathways batch inference workloads, where latency is crucial.
+The 'colocate' mode deploys the "pathways-head" pod besides a "worker" pod on one of the TPU workers. This is preferred for Pathways batch inference workloads, where latency is crucial.
```

Comment:
I understand the meaning of `colocate` here given the LWS context I have, but I think this could easily be confused with colocated python. I'd prefer if we reused the LeaderWorkerSet language. I'm assuming this mode relies on the LWS abstractions, please correct me if I'm wrong.

Reply:
Thanks for raising this. The context here is the deployment mode of the controller (pathways head pod) with respect to the Pathways workers. I can understand the confusion when thinking from the perspective of remote python, where colocated means something else. Let's work towards adding more context on the terminology in the README in upcoming PRs.